charter.md

```diff
@@ -0,0 +1,55 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established. By default, the group will operate in public. However, specific decision-making processes can be made member-confidential by defining them as such in a process document published with the consensus of the team.
+
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. The term limit for the chair will be 1 year and the same individual can be reappointed as chair, if there is consensus. If the team is unable to meet due to absence of a chair, the Core Team may appoint one directly.
+
+Meeting time and agenda should be announced by the chair at least 48 hours in advance, and meeting minutes posted in the repository.
+
+The Interop Team's work is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+All contributions are licensed under the terms in [CONTRIBUTING.md](https://github.com/web-platform-tests/wpt/blob/master/CONTRIBUTING.md).
+
+Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
```

Comment:
```suggestion
Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs). Existing members of the team are notified that when such an RFC is created.
```

Do we believe the one-week rule for RFCs suffices? When the team has previously had biweekly meetings it was at times the case people didn't look at issues more often than the meetings, so an RFC could be filed and approved between two meetings without all those active in the Interop Team being aware of it.

This is also notably different to the W3C where re-chartering (including any substantive changes) requires rejoining the group—it seems undesirable that we might end up with members of the group who don't agree with the charter after a substantive change has been made.

Reply:
I think I agree in general that the RFC period as specified is rather short for cases where there's non-trivial disagreement and that the process we use in practice allows more time to provide feedback. The escalation process as written has never been used, and it's clear that in cases like this it would not actually work.

Maybe we should also look to update that to reflect reality better.

Reply:
I think we'd need to change the RFC process to fix this, and this charter can't add rules to the RFC process. What I'd like to happen in practice is that changes are run by the Interop Team and that we're given at least 2 weeks to react.

I'm happy to just apply the suggested working, but @gsnedders @jgraham would that be good enough for you?

Reply:
Here's a wording change I think does the job:

```suggestion
Substantive changes to this charter require the consensus of the Interop Team and should go through the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
```

In practice, the Interop Team would first internally agree to the change, and then put it through the RFC process.

Reply:
I don't think it can use 'should'. Maybe: "Substantive changes to this charter can be made using the wpt RFC process, with an additional requirement of explicit consensus from the Interop Team participants."

Reply:
Alright, so:

```suggestion
Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs). Substantive changes to this charter can be made using the wpt RFC process, with an additional requirement of explicit consensus from the Interop Team participants.
```